tidb-binlog/pump_client/bench_test.go

```diff
@@ -21,6 +21,7 @@ import (
 	"time"
 
 	pb "github.com/pingcap/tipb/go-binlog"
+	"github.com/stretchr/testify/assert"
```

Comment:
```suggestion
	"github.com/stretchr/testify/require"
```

Almost we prefer require to fail fast.

Reply:
sure,i accept it.

Reply:
I have resubmitted the code